deploy/pvc.yaml

```diff
@@ -7,5 +7,5 @@ spec:
    - ReadWriteOnce
   resources:
     requests:
-      storage: 64Mi
+      storage: 1Gi
```

Comment:
Why do we need to create such a big volume? this implies we MUST have 1Gi everywhere. If 64Mi is not suffucient, would 128Mi do?

Reply:
This change was made so that we could use the scripts in the deploy directory to do what is described in quick.md leading to less confusion for people trying out Mayastor.
I will revert, or change quick.md.

Reply:
Reverted.